sdk/aqueduct/backend/api_client.py

```diff
@@ -40,6 +41,8 @@
     SavedObjectUpdate,
 )
 
+MAX_REQUEST_BODY_SIZE = 32 << 20
```

Comment:
Please document this. What's are the units? What is the human-understandable value?